scrapbook/api.py

```diff
@@ -67,7 +67,14 @@ def glue(name, data, encoder=None, display=None):
 
     # TODO: Implement the cool stuff. Remote storage indicators?!? Maybe remote media type?!?
     if not encoder:
-        encoder = encoder_registry.determine_encoder_name(data)
+        try:
+            encoder = encoder_registry.determine_encoder_name(data)
+        except NotImplementedError:
+            if display is not None:
+                # The scrap isn't encodable, but the display object it might be, so don't raise
```

Comment:
Nit: this code comment is a little unclear. Maybe try this.

> Do not raise an exception if the scrap isn't encodable because the corresponding display object may be.